src/SFA.DAS.Campaign.Web/Views/Apprentice/WhatAreTheBenefitsToMe.cshtml

```diff
@@ -33,7 +33,7 @@
     Your salary will depend upon the industry, location and type of apprenticeship you choose.
 </p>
 <p>
-    The minimum wage for an apprentice is £3.90 per hour if you’re aged 16 to 18 - but many employers pay far more than this.
+    If you’re aged 16 to 18 or in the first year of your apprenticeship, you’re entitled to the apprentice rate. This is currently £4.15 - but many employers pay more than this.
```

Comment:
I think you might have some magic single quotes in there.  Happens with a copy and paste content.

Reply:
I have updated all occurrences